ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -642,5 +657,19 @@ private void UpdateLayout(DependencyObject viewToUpdate, IViewManager viewManage
                 viewManager.SetDimensions(viewToUpdate, dimensions);
             }
         }
+
+        private void AssertOnCorrectDispatcher()
+        {
+#if WINDOWS_UWP
+#if DEBUG
+            if (!DispatcherHelpers.IsOnDispatcher(Dispatcher))
+            {
+                throw new InvalidOperationException("Thread does not have correct dispatcher access.");
```

Comment:
It is a little bit unclear what is considered the "correct dispatcher", a comment may help